package.json

```diff
@@ -33,7 +33,18 @@
       "transform-object-rest-spread"
     ],
     "presets": [
-      "es2015"
+      [
+        "env",
+        {
+          "targets": {
+            "browsers": [
+              "last 3 Chrome versions",
+              "last 3 Firefox versions",
+              "last 3 Opera versions"
```

Comment:
Ideally we could specify the browser according to the current build target, but maybe a bit over engineered for our case